source/includes/_tipser-elements.md

```diff
@@ -236,4 +236,19 @@ for products on sale.
     z-index: 10;
 }
 ```
-The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
\ No newline at end of file
+The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
+<br><br><br><br><br>
+
+### Primary color ###
+```css
+.te-primary-fill {
+    background-color: #5F9F9F;
```

Comment:
shouldn't we have something like `te-primary-text` then ? I think it is important to have control over a text color used on primary background.

Reply:
`te-primary-text` would mean that we assign the primary color to the text. Makes no sense with the primary color as background-color.
But I'll add a warning to the docs that sometimes the `color: sth` needs to be changed to a contrasting one.